test/functional/feature_notifications.py

```diff
@@ -29,7 +29,8 @@ def setup_network(self):
                             "-blocknotify=echo > {}".format(os.path.join(self.blocknotify_dir, '%s'))],
                            ["-blockversion=211",
                             "-rescan",
-                            "-walletnotify=echo > {}".format(os.path.join(self.walletnotify_dir, '%s'))]]
+                            "-wallet=t- 1",
```

Comment:
This test fails when the wallet name has a `'` in it. Probably need to quote it below?